empyrical/stats.py

```diff
@@ -739,11 +976,11 @@ def stability_of_timeseries(returns):
     if len(returns) < 2:
         return np.nan
 
-    returns = returns.dropna()
+    returns = returns[~np.isnan(returns)]
```

Comment:
On my computer, `dropna` has better performance than this other method, is there a reason for changing this line?